app/views/did_you_know_alert_mailer/email.html.haml

```diff
@@ -0,0 +1,19 @@
+%link{rel: 'stylesheet', href:'/mailer.css'}
+%p.paragraph
+  Hello,
+  %p.paragraph
+    The following article being worked on as a part of your course has received a Did You Know nomination
+    %a.link{href: "https://upload.wikimedia.org/wikipedia/commons/e/ee/Classroom_handout_-_Submitting_an_article_to_the_Did_You_Know_process.pdf"}= "#{"(PDF handout)"}: "
```

Comment:
Just "PDF handout" should be a link, not the parens or colon.

Looking at the new screenshot, I think it will be better to reorganize the text a little so that it's more clear that the article is the important part, and the PDF is for context. I suggest:

> An article that is being worked on as part of your course, `<article link>`, has received a Did You Know nomination. See the [PDF handout about the Did You Know process](https://upload.wikimedia.org/wikipedia/commons/e/ee/Classroom_handout_-_Submitting_an_article_to_the_Did_You_Know_process.pdf) for more details.

Reply:
ok